sentry-core/src/main/java/io/sentry/core/Sentry.java

```diff
@@ -238,7 +289,8 @@ public static synchronized void close() {
    * @param breadcrumb the breadcrumb
    * @param hint SDK specific but provides high level information about the origin of the event
    */
-  public static void addBreadcrumb(@NotNull Breadcrumb breadcrumb, @Nullable Object hint) {
+  public static void addBreadcrumb(
```

Comment:
Super hard to review this PR with so much unrelated changes like this

Reply:
true, I should not have changed this, but it's a Draft right